db/migrate/20200508150740_remove_vmdb_database_tables.rb

```diff
@@ -0,0 +1,78 @@
+class RemoveVmdbDatabaseTables < ActiveRecord::Migration[5.2]
+  def up
+    drop_table :vmdb_database_metrics
+    drop_table :vmdb_databases
+    drop_table :vmdb_indexes
+    drop_table :vmdb_metrics
+    drop_table :vmdb_tables
+  end
+
+  def down
+    create_table "vmdb_database_metrics", force: :cascade do |t|
```

Comment:
I grabbed the following from my schem.rb for the tables that were deleted.  I think that's good enough for a down migration.